### PR TITLE
[TK-6]: build login screen

### PR DESCRIPTION
**Pull request by Alisheikh6**
## Jira: [TK-6](https://shop-go.atlassian.net/jira/software/projects/TK/boards/2?selectedIssue=TK-6/)

**Description**

- UI only story
- Autolayout
- Buttons for social media log in
- corner radius for buttons is 22

**How to test it**
- Navigate onto the Login View Controller screen
- Click log in or social media buttons

#Screenshots&Gifs:

### CODE REVIEW

File: Vape-X/LoginViewController/LoginVC.swift [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+//  Vape-X
+//
+//  Created by sheikh on 9/21/22.
+//
+
+import UIKit
+
+class LoginVC: UIViewController {
+    
+    //Outlets
```
Comment (haroonmaqsoodch) [inline Vape-X/LoginViewController/LoginVC.swift:12]: remove this comment and the space between the outlets

File: Vape-X/LoginViewController/LoginVC.swift [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    @IBOutlet weak var txtFieldUsername: UITextField!
```
Comment (haroonmaqsoodch) [inline Vape-X/LoginViewController/LoginVC.swift:13]: fix the name of the textfield

File: Vape-X/LoginViewController/LoginVC.swift [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    
+    @IBOutlet weak var passwordTxtField: UITextField!
```
Comment (haroonmaqsoodch) [inline Vape-X/LoginViewController/LoginVC.swift:15]: fix the name

File: Vape-X/LoginViewController/LoginVC.swift [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+    
+    //Actions
+    @IBAction func passwordBtnPressed(_ sender: UIButton) {
+    }
```
Comment (haroonmaqsoodch) [inline Vape-X/LoginViewController/LoginVC.swift:19]: fix the names of all these outlets and actions, don't use short forms

File: Vape-X/LoginViewController/LoginVC.swift [hunk 5]
```diff
@@ -0,0 +1,53 @@
+//
+//  LoginVC.swift
+//  Vape-X
+//
+//  Created by sheikh on 9/21/22.
+//
+
+import UIKit
+
+class LoginVC: UIViewController {
```
Comment (haroonmaqsoodch) [inline Vape-X/LoginViewController/LoginVC.swift:10]: write LoginViewController instead of LoginVC

File: Vape-X/Base.lproj/Main.storyboard [hunk 6]
```diff
@@ -1,24 +1,50 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="13122.16" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="BYZ-38-t0r">
+<document type="com.apple.InterfaceBuilder3.CocoaTouch.Storyboard.XIB" version="3.0" toolsVersion="21225" targetRuntime="iOS.CocoaTouch" propertyAccessControl="none" useAutolayout="YES" useTraitCollections="YES" useSafeAreas="YES" colorMatched="YES" initialViewController="x89-rd-hdC">
+    <device id="retina6_0" orientation="portrait" appearance="light"/>
```
Comment (haroonmaqsoodch) [inline Vape-X/Base.lproj/Main.storyboard:3]: make a separate storyboard file and move your changes there

**Review by haroonmaqsoodch — COMMENTED** [review]
corner radius isn't 22

File: Vape-X/LoginViewController/LoginVC.swift [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+    }
+    
+    
+    @IBAction func igBtnPressed(_ sender: Any) {
+    }
+    
+    @IBAction func twtBtnPressed(_ sender: Any) {
+    }
+    
+
```
Comment (MuhammadHuzaifa2) [inline Vape-X/LoginViewController/LoginVC.swift:34]: Remove unnecessary spaces

**Comment by basitalich** [comment]
resolve the conflicts.

File: Vape-X/LoginViewController/LoginVC.swift (hunk 1 above)
Comment (Alisheikh6) [reply Vape-X/LoginViewController/LoginVC.swift]: ok

File: Vape-X/LoginViewController/LoginVC.swift (hunk 2 above)
Comment (Alisheikh6) [reply Vape-X/LoginViewController/LoginVC.swift]: ok

File: Vape-X/LoginViewController/LoginVC.swift (hunk 3 above)
Comment (Alisheikh6) [reply Vape-X/LoginViewController/LoginVC.swift]: ok

File: Vape-X/LoginViewController/LoginVC.swift (hunk 4 above)
Comment (Alisheikh6) [reply Vape-X/LoginViewController/LoginVC.swift:15]: ok

File: Vape-X/LoginViewController/LoginVC.swift (hunk 5 above)
Comment (Alisheikh6) [reply Vape-X/LoginViewController/LoginVC.swift]: ok will update name

File: Vape-X/LoginViewController/LoginVC.swift (hunk 7 above)
Comment (Alisheikh6) [reply Vape-X/LoginViewController/LoginVC.swift]: ok

File: Vape-X/Base.lproj/Main.storyboard [hunk 8]
```diff
@@ -1,24 +1,24 @@
...
     <dependencies>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="13104.12"/>
-        <capability name="Safe area layout guides" minToolsVersion="9.0"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="21207"/>
         <capability name="documents saved in the Xcode 8 format" minToolsVersion="8.0"/>
     </dependencies>
     <scenes>
-        <!--View Controller-->
-        <scene sceneID="tne-QT-ifu">
+        <!--Navigation Controller-->
```
Comment (ammadahmed22) [inline Vape-X/Base.lproj/Main.storyboard:9]: this is a navigation controller  this supposed to be a view controller

**Review by ammadahmed22 — COMMENTED** [review]
your safe areas aren't covered from the top of the phone on different simulators

File: Vape-X/Base.lproj/Main.storyboard (hunk 6 above)
Comment (Alisheikh6) [reply Vape-X/Base.lproj/Main.storyboard:3]: Fixed

File: Vape-X/LoginViewController/LoginVC.swift (hunk 7 above)
Comment (Alisheikh6) [reply Vape-X/LoginViewController/LoginVC.swift]: fixed

File: Vape-X/Base.lproj/Main.storyboard (hunk 8 above)
Comment (Alisheikh6) [reply Vape-X/Base.lproj/Main.storyboard]: Fixed

**Review by basitalich — APPROVED** [review]

**Pull request merged**